InstagramStories/Source/StoryPreview/IGStoryPreviewController.swift

```diff
@@ -147,6 +148,18 @@ final class IGStoryPreviewController: UIViewController, UIGestureRecognizerDeleg
     @objc func didSwipeDown(_ sender: Any) {
         dismiss(animated: true, completion: nil)
     }
+ 
+   private func setup_viewConstraint() {
+        view.addSubview(_view)
+        _view.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            _view.leftAnchor.constraint(equalTo: view.igLeftAnchor,constant: .zero),
+            _view.topAnchor.constraint(equalTo: view.igTopAnchor,constant: .zero),
+            _view.rightAnchor.constraint(equalTo: view.igRightAnchor,constant: .zero),
+            _view.bottomAnchor.constraint(equalTo: view.bottomAnchor,constant: .zero)
```

Comment:
use `view.igBottomAnchor` instead of `view.bottomAnchor`
change into this
_view.bottomAnchor.constraint(equalTo: view.igBottomAnchor,constant: .zero)